content/en/docs/concepts/cluster-administration/events.md

```diff
@@ -0,0 +1,34 @@
+---
+reviewers:
+- piosz
+- x13n
+content_template: templates/concept
+title: Events in Kubernetes
+---
+
+{{% capture overview %}}
+
+Kubernetes events are objects that provide insight into what is happening
```

Comment:
Consider using a glossary definition shortcode here, that automatically includes either the short or full version of the glossary text for this object.

Reply:
done